docs/dev/release.md

```diff
@@ -142,12 +142,18 @@ The direct link, at the time of writing is https://connect.redhat.com/component/
 Copy the **certified** image SHAs of the **amd64** and the **arm64** image:
 
 ![img.png](certified-image-sha.png)
-
+> **NOTE**
+> 
+> In case you can't see the images tags in the https://connect.redhat.com go to the https://quay.io/repository/mongodb/mongodb-atlas-kubernetes-operator?tab=tags
+and select the latest released tag that ends with `-certified` e.g. `2.4.0-certified`. Then click on `manifest` link for the selected image.
+You should see two available manifests for `amd64` and `arm64` that you can use. Here is the example: https://quay.io/repository/mongodb/mongodb-atlas-kubernetes-operator/manifest/sha256:20740f06db3b6e768b40562ee50c729aba9870a9051045b4fc49d0c404cc18f6
+![img.png](img.png)
```

Comment:
img.png does not seem to be part of this PR

Reply:
+1

Reply:
Omg, missed that. Fixed!